leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/Fragments.kt

```diff
@@ -0,0 +1,49 @@
+package com.squareup.leakcanary.tests
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.fragment.app.FragmentManager
+import java.util.concurrent.CountDownLatch
+
+class Fragments private constructor() {
```

Comment:
For this class, you would want to convert the `waitForFragmentDetached` and `waitForFragmentViewDestroyed` into extension functions since these are effectively helper functions for `FragmentActivity`

Something like `internal fun FragmentActivity.waitForFragmentViewDestroyed(): CountDownLatch()`

Reply:
Changed, good idea!